tests/test_netsuite/test_models.py

```
@@ -363,11 +364,12 @@
 
     credit_card = CreditCardCharge.create_credit_card_charge(expense_group)
     configuration = Configuration.objects.get(workspace_id=1)
-    credit_card_charge_lineitem = CreditCardChargeLineItem.create_credit_card_charge_lineitem(expense_group, configuration)
+    credit_card_charge_lineitems = CreditCardChargeLineItem.create_credit_card_charge_lineitems(expense_group, configuration)
```

Ruff (0.7.0)

`CreditCardChargeLineItem` may be undefined, or defined from star imports

(F405)